integration_test/remove_disks_test.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright © 2021 Cisco and/or its affiliates. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration_test
+
+import (
+	"github.com/banzaicloud/go-cruise-control/integration_test/helpers"
+	"github.com/banzaicloud/go-cruise-control/pkg/api"
+	"github.com/banzaicloud/go-cruise-control/pkg/types"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Remove Disks",
+	Label("api:remove_disks", "api:user_tasks", "api:kafka_cluster_load", "api:state"),
+	Serial,
+	func() {
+		const (
+			brokerID = 0
+			logDir   = "/var/lib/kafka/data0"
+		)
+
+		BeforeEach(func(ctx SpecContext) {
+			By("waiting until Cruise Control is ready")
+			Eventually(ctx, func() bool {
+				ready, err := helpers.IsCruiseControlReady(ctx, cruisecontrol)
+				Expect(err).NotTo(HaveOccurred())
+				return ready
+			}, CruiseControlReadyTimeout, 15).Should(BeTrue())
+		})
+
+		Describe("Removing a disk in Kafka cluster", func() {
+			It("should return no error", func(ctx SpecContext) {
+				By("sending a remove request to Cruise Control")
+				req := &api.RemoveDisksRequest{}
+
+				req.BrokerIDAndLogDirs = map[int32][]string{
+					brokerID: {logDir},
+				}
+				req.Reason = "integration testing"
+
+				resp, err := cruisecontrol.RemoveDisks(ctx, req)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(resp.Failed()).To(BeFalse())
+
+				By("waiting until the remove task finished")
+				Eventually(ctx, func() bool {
+					finished, err := helpers.HasUserTaskFinished(ctx, cruisecontrol, resp.TaskID)
+					Expect(err).NotTo(HaveOccurred())
+					return finished
+				}, 600, 15).Should(BeTrue())
```

Comment:
what does 600 and 15 represent? Would it make sense to make them constants (with a descriptive name)?

Reply:
I have made these numbers constants with a descriptive name. 
600 = timeout
15 = pollInterval

(unfortunately magic numbers are also used across other tests)